wiki/Modding_Guides/How_to_not_Embarrass_Yourself_in_the_Modding_Queues_Subforum/en.md

```diff
@@ -1,9 +1,12 @@
+How to not Embarrass Yourself in the Modding Queues Subforum
+=============================================================
+
 So osu!forums has this subforum as many know as **[Modding Queues](https://osu.ppy.sh/forum/60)**. It might seem intimidating at first but here are the tips and tricks on **How to not Embarrass Yourself in the Modding Queues Subforum**.
 
 Do NOT Abuse This Subforum
 --------------------------
 
-[*Do NOT Abuse This Subforum* by awp](https://osu.ppy.sh/forum/t/21552)
+[*Do __not__ Abuse This Subforum* by awp](https://osu.ppy.sh/forum/t/21552)
```

Comment:
since this is the title of that thread, leave as-is...? imo, this is kinda like a direct quote, so we shouldn't modify much of the spelling/grammar/caps